install/mysql/glpi-empty.sql

```diff
@@ -1249,7 +1241,7 @@ CREATE TABLE `glpi_impactitems` (
 DROP TABLE IF EXISTS `glpi_impactcontexts`;
 CREATE TABLE `glpi_impactcontexts` (
   `id` INT NOT NULL AUTO_INCREMENT,
-  `positions` TEXT NOT NULL,
```

Comment:
I do not understand those ones. All other types seems to be uppercase.

Reply:
Types are mixed upper and lowercase. Changing this case was a test for my regex. I didn't revert it afterward sorry.

Reply:
If my regex is right : 

```
grep "\`[[:alnum:]_]\+\` [a-z]\+" install/mysql/glpi-empty.sql | wc -l
grep "\`[[:alnum:]_]\+\` [A-Z]\+" install/mysql/glpi-empty.sql | wc -l
```
- nb of lowercase types : 3613
- nb of uppercase types : 66

:open_mouth:

Reply:
IMHO, as `SHOW CREATE TABLE` seems to always return lowercased types, changing these 66 occurences of uppercased types is an acceptable change (and will not be so hard to validate).

Reply:
done in 89e501aa2385b85e942432c72c70df0da1cabd25